Fix failing test still using method="form-data-post"

```python
======================================================================
FAIL: test_values_without_whitespaces_are_processed_successfully (pyxform.tests_v1.test_whitespace.WhitespaceTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/circleci/work/pyxform/tests_v1/test_whitespace.py", line 34, in test_values_without_whitespaces_are_processed_successfully
    self.assertEqual(1, xml.count(expected))
AssertionError: 1 != 0
```

--- a/pyxform/tests_v1/test_whitespace.py
+++ b/pyxform/tests_v1/test_whitespace.py
@@ -29,7 +29,7 @@ class WhitespaceTest(PyxformTestCase):
           """
 
         survey = self.md_to_pyxform_survey(md_raw=md)
-        expected = """<submission action="https://odk.ona.io/random_person/submission" base64RsaPublicKey="MIIB" method="form-data-post"/>"""
+        expected = """<submission action="https://odk.ona.io/random_person/submission" base64RsaPublicKey="MIIB" method="post"/>"""
         xml = survey._to_pretty_xml()
         self.assertEqual(1, xml.count(expected))
         self.assertPyxformXform(md=md, xml__contains=expected, run_odk_validate=True)

